app/indexers/hyrax/file_set_indexer.rb

```diff
@@ -1,57 +1,115 @@
+        # Metadata from the FileSet
+        solr_doc['file_ids_ssim']         = resource.file_ids&.map(&:to_s)
+        solr_doc['original_file_id_ssi']  = resource.original_file_id.to_s
+        solr_doc['thumbnail_id_ssi']      = resource.thumbnail_id.to_s
+        solr_doc['extracted_text_id_ssi'] = resource.extracted_text_id.to_s
+
+        # Add in metadata from the original file.
+        file_metadata = original_file
+        return solr_doc unless file_metadata
+
         # Label is the actual file name. It's not editable by the user.
-        solr_doc['label_tesim'] = object.label
-        solr_doc['label_ssi']   = object.label
-        solr_doc['file_format_tesim'] = file_format
-        solr_doc['file_format_sim']   = file_format
-        solr_doc['file_size_lts'] = object.file_size[0]
-        solr_doc['all_text_timv'] = object.extracted_text.content if object.extracted_text.present?
-        solr_doc['height_is'] = Integer(object.height.first) if object.height.present?
-        solr_doc['width_is']  = Integer(object.width.first) if object.width.present?
-        solr_doc['visibility_ssi'] = object.visibility
-        solr_doc['mime_type_ssi']  = object.mime_type
-        # Index the Fedora-generated SHA1 digest to create a linkage between
-        # files on disk (in fcrepo.binary-store-path) and objects in the repository.
-        solr_doc['digest_ssim']             = [digest_from_content]
-        solr_doc['page_count_tesim']        = object.page_count
-        solr_doc['file_title_tesim']        = object.file_title
-        solr_doc['duration_tesim']          = object.duration
-        solr_doc['sample_rate_tesim']       = object.sample_rate
-        solr_doc['original_checksum_tesim'] = object.original_checksum
-        solr_doc['alpha_channels_ssi']      = object.alpha_channels
-        solr_doc['original_file_id_ssi']    = original_file_id
+        solr_doc['original_file_alternate_ids_tesim'] = file_metadata.alternate_ids if file_metadata.alternate_ids.present?
+
+        solr_doc['original_filename_tesi']  = file_metadata.original_filename if file_metadata.original_filename.present?
+        solr_doc['original_filename_ssi']   = file_metadata.original_filename if file_metadata.original_filename.present?
+        solr_doc['mime_type_tesi']          = file_metadata.mime_type if file_metadata.mime_type.present?
+        solr_doc['mime_type_ssi']           = file_metadata.mime_type if file_metadata.mime_type.present?
+
+        solr_doc['file_format_tesim']       = file_format(file_metadata)
+        solr_doc['file_format_sim']         = file_format(file_metadata)
+        solr_doc['file_size_lts']           = file_metadata.size[0]
+        solr_doc['type_tesim']              = file_metadata.type if file_metadata.type.present?
+
+        # attributes set by fits
+        solr_doc['format_label_tesim']      = file_metadata.format_label if file_metadata.format_label.present?
```

Comment:
Do we want to include all file metadata in the solr_doc?  Right now, that is what this does.